src/about.html

```diff
@@ -0,0 +1,287 @@
+					</div>
+					<div class="container mt-5 mb-5">
+						<div class="row">
+							<div class="col-lg-12">
+								<ul class="timeline">
+									<li>
+										<h1 class="headline">Quarter 1</h1>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">2023 HR Ministry Network Conference</span></h2>
+										<p>The HR Ministry Network Launched with a Conference tailored for HR and Finance Ministry Professionals. Get access to the sessions at https://Kingdomone.Academy</p>
+									</li>
+									<li>
+										<h1 class="headline">Quarter 2</h1>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">Kingdom One Retirement 403(b)9</span></h2>
+										<p>Retirement is a key aspect of Total Rewards and Staff Compensation. We offer a 403(b)9 so your staff can retire in comfort. https://www.kingdomone.co/k1-403b9</p>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">Spark Compensation Survey</span></h2>
+										<p>Spark Compensation Survey is here to help ministry recruiters and finance teams build compensation philosophies that work. https://kingdomone.co/compensation</p>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">Leadership Development - Coming Soon</span></h2>
+										<p>In life and ministry, character matters. This leadership developments incorporates character into performance and culture.</p>
+									</li>
+									<li>
+										<h1 class="headline">Quarter 3</h1>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">Spark Staffing 2.0</span></h2>
+										<p>Your favorite Ministry Staffing site is getting some upgrades. https://sparkstaffing.co</p>
+									</li>
+									<li>
+										<h2 class="subheadline"><span class="text-primary">Legal Update for 2024 - Coming Soon</span></h2>
+										<p>We keep you in the loop for California 2024 legal updates that impact your ministry.</p>
+									</li>
+								</ul>
+							</div>
+						</div>
+					</div>
+					 
+				</div>
+			</div>
+		</section>
+		<section class="values">
+			<div class="values__background">
+				<div class="values__background--lower"></div>
+				<div class="values__background--upper"></div>
+			</div>
+			<div class="container values__content">
+				<div class="row">
+					<div class="col justify-content-center align-items-center">
+						<h1 class="headline text-primary leaf-adjustment-right">our values<img src="/src/assets/images/rightorientationgreenleaves.png"></h1>
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-3">
+						<img src="/src/assets/images/relationship.png">
+					</div>
+					<div class="col justify-content-center align-items-stretch">
```

Comment:
in bootstrap, `col`s aren't set with `display:flex` so you'll need to add that each time you want to use it